src/cascadia/LocalTests_SettingsModel/DeserializationTests.cpp

```diff
@@ -1900,15 +1899,6 @@ namespace SettingsModelLocalTests
         verifyEmptyPD(missingPDJson);
     }
 
-    void DeserializationTests::TestValidDefaults()
-    {
-        // GH#8146: A LoadDefaults call should populate the list of active profiles
-
-        const auto settings{ CascadiaSettings::LoadDefaults() };
-        VERIFY_ARE_EQUAL(settings.ActiveProfiles().Size(), settings.AllProfiles().Size());
-        VERIFY_ARE_EQUAL(settings.AllProfiles().Size(), 2u);
-    }

```

Comment:
Would it be valuable to still have a test that just straight-up deserializes the defaults.json? On one hand, I like the idea of ensuring that defaults.json is valid. On the other hand, we've had a bad track record of looking at our local tests. But I'm concerned that we could somehow break defaults.json and never notice? Is that even possible tbh? haha